### ISSUE TITLE
Undefined symbols for architecture x86_64 TVIVideoClient, TVILocalMedia

### ISSUE THREAD

**Issue by mfkrause** [blackuy/react-native-twilio-video-webrtc#216]
### Steps to reproduce
1. Install the node package
2. Import everything
3. Run the example app

I'm using Expo, therefore not using CocoaPods or the native *.xcodeproj environment etc. at all, therefore I only installed the node package and imported it. I suppose the error comes from this since it's missing some kind of configuration. Kinda new to the React.js / React Native environment, so I guess I'm missing some obvious here.

### Actual behaviour
I get the error:
Invariant Violation: Native module cannot be null.
This error is located at:
in TwilioVideo

### Environment
- **Node.js version**: v10.15.1
- **React Native version**: 0.59.8
- **React Native platform + platform version**: iOS 12

### react-native-twilio-video-webrtc
**Version**: 1.0.1

**Comment by mfkrause** [comment]
Okay, at this point I'm close to giving up.
First off, Expo does not support WebRTC at all. One needs to detach / eject from Expo in order to use the native modules needed for WebRTC support. I did eject. Still, I cannot get this to run at all. Without react-native-twilio-video-webrtc, the project runs fine from my *.xcworkspace.
I added the pod 'TwilioVideo' to my Podfile and installed it. I don't have any "libRNTwilioVideoWebRTC.a" file. No where to be found. When building the project, it's telling me TVILocalMedia and TVIVideoClient are unknown type names.

What should I do now? I even looked up the libRNTwilioVideoWebRTC.a file in this repo, I cannot find it anywhere.
